src/settings/JobProfiles/JobProfileDetailsRoute/JobProfileDetailsRoute.js

```diff
@@ -5,7 +5,6 @@ import { useQuery } from 'react-query';
 import { useOkapiKy } from '@folio/stripes/core';
 
 import { JobProfileDetails } from '../JobProfileDetails';
-import { DEFAULT_JOB_PROFILE_ID } from '../../../utils';
```

Comment:
Can we remove DEFAULT_JOB_PROFILE_ID drom utils, if it not used anymore?

Reply:
@uladislausamets it's removed already.

Reply:
I mean remove from https://github.com/folio-org/ui-data-export/blob/367f83f8e683fb6824582caead2e2e9726c93c86/src/utils/constants.js

Reply:
@uladislausamets I fixed that. Thanks a lot